Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -272,3 +277,89 @@ void ProjectData::parseOutput(ConfigTree const& output_config,
 
 	_output_file_prefix = path + *file;
 }
+
+void ProjectData::parseTimeStepping(ConfigTree const& timestepping_config)
+{
+	using namespace ProcessLib;
+
+	DBUG("Reading timestepping configuration.");
+
+	auto const type = timestepping_config.get_optional<std::string>("type");
+	if (!type)
+	{
+		ERR("Could not find required timestepper type.");
+		std::abort();
+	}
+
+	if (*type == "FixedTimeStepping")
+	{
+		_time_stepper.reset(NumLib::FixedTimeStepping::newInstance(timestepping_config));
+	}
+	else if (*type == "SingleStep")
+	{
+		_time_stepper.reset(new NumLib::FixedTimeStepping(0.0, 1.0, 1.0));
+	}
+	else
+	{
+		ERR("Unknown timestepper type: `%s'.", type->c_str());
+		std::abort();
+	}
+
+	if (!_time_stepper)
+	{
+		ERR("Initialization of timestepper failed.");
+		std::abort();
+	}
+}
+
+void ProjectData::initializeProcesses()
+{
+	INFO("Initialize processes.");
+
+	for (auto& p : _processes)
+	{
+		p->initialize();
+	}
+}
+
+void ProjectData::solveProcesses()
+{
+	INFO("Solve processes.");
+
+	std::string const out_pref = getOutputFilePrefix();
+	std::string const output_file_name(getOutputFilePrefix() + ".vtu");
+
+    while (_time_stepper->next()) // skips zeroth timestep, but OK since end of first timestep is after first delta t
```

Comment:
Just an idea:

if limiting number of time steps, then this could become a for loop.
```c++
std::size_t const maximum_timesteps = 42; // zero for unlimited.
for (std::size_t timestep = 1; timestep < maxium_timesteps; ++i)
{
    ...
}
```
but maybe there is already a mechanism for this limit inside the time_stepper objects...